checkpoints/checkpoint1/index.html

```diff
@@ -0,0 +1,255 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>MUBI homepage</title>
+  <link rel="stylesheet" type="text/css" href="http://yui.yahooapis.com/3.18.1/build/cssreset/cssreset-min.css">
```

Comment:
missing a reset style css

Reply:
Is line 5 not a reset stylesheet? =/

Reply:
I am sorry, you are correct it is a reset.